pkg/test/framework/components/istio/operator.go

```diff
@@ -693,3 +714,132 @@ func deployCACerts(workDir string, env *kube.Environment, cfg Config) error {
 	}
 	return nil
 }
+
+func createIstiodEndpoint(discoveryAddress string, cfg Config, cluster resource.Cluster) error {
```

Comment:
Can we rename this to be a little more clear ... maybe something like `configureDiscoveryForRemoteCluster`?

Reply:
fxied